tools/scripts/generate_mocks.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+set -o errexit
+set -ox nounset
+
+MOCK_FOLDER=${MOCK_FOLDER:-../admin}
```

Comment:
I used moq only as example. We do not have strong opinion. 
@lantoli I think we might want to change to mockery?

Reply:
yes, that's our proposal for Terraform project